docs/database.md

```diff
@@ -0,0 +1,123 @@
+| `binary_blob` | `binary`                  | FALSE    |          -           | Attachment file data as binary         |
+| `created_at`  | `dateTime`                | FALSE    |       `now()`        | When the row was first created         |
+
+#### Indexes
+
+| columns | Index Type | description |
+| :------ | :--------- | :---------- |
+| `id`    | PRIMARY    | Primary key |
+
+### `demand`
+
+| column                     | PostgreSQL type | nullable |       default        | description                               |
+| :------------------------- | :-------------- | :------- | :------------------: | :---------------------------------------- |
+| `id`                       | `UUID`          | FALSE    | `uuid_generate_v4()` | Unique identifier for the `attachment`    |
+| `owner`                    | `STRING (48)`   | FALSE    |          -           | Demand owner name                         |
+| `subtype`                  | `ENUM`          | FALSE    |          -           | The demand subtype (`order`, `capacity`)  |
+| `state`                    | `ENUM`          | FALSE    |          -           | The demand state (`created`, `allocated`) |
+| `parameters_attachment_id` | `UUID`          | FALSE    |          -           | The ID of the associated attachment       |
+| `latest_token_id`          | `INT`           | TRUE     |          -           | Possible current token ID                 |
+| `original_token_id`        | `INT`           | TRUE     |          -           | Possible original token ID                |
+| `created_at`               | `dateTime`      | FALSE    |       `now()`        | When the row was first created            |
+| `updated_at`               | `dateTime`      | FALSE    |       `now()`        | When the row was last updated             |
+
+#### Indexes
+
+| columns | Index Type | description |
+| :------ | :--------- | :---------- |
+| `id`    | PRIMARY    | Primary key |
+
+#### Foreign Keys
+
+| columns                    | References     | description                  |
+| :------------------------- | :------------- | :--------------------------- |
+| `parameters_attachment_id` | attachment(id) | The id of the attachment row |
+
+### `transaction`
+
+| column             | PostgreSQL type | nullable |       default        | description                                             |
+| :----------------- | :-------------- | :------- | :------------------: | :------------------------------------------------------ |
+| `id`               | `UUID`          | FALSE    | `uuid_generate_v4()` | Unique identifier for the `attachment`                  |
+| `local_id`         | `UUID`          | FALSE    |                      | The Match2 id of the transaction                        |
+| `api_type`         | `ENUM`          | FALSE    |                      | The type of transaction (`match2`, `order`, `capacity`) |
```

Comment:
maybe the 'entity' of the transaction might be a better definition? Type of transaction is the next row